tigers-lottery/app/src/main/java/com/example/tigers_lottery/models/User.java

```diff
@@ -143,10 +143,10 @@ public void setUserId(Object userId) {
     @PropertyName("DOB")
     public void setDateOfBirth(Timestamp dateOfBirth) { this.dateOfBirth = dateOfBirth; }
 
-    @PropertyName("facility-location")
+    @PropertyName("facility_location")
     public String getFacilityLocation() { return facilityLocation; }
 
-    @PropertyName("facility-location")
+    @PropertyName("facility_location")
```

Comment:
Oh shit, it was meant to be "facility_location" but abhro made a mistake and did "facility-location" so I had to leave it as "facility-location". Good catch An but lets revert this change at least until we clean the DB